chaos-workers/chaos-experiments/scripts/utils.sh

```diff
@@ -38,7 +38,7 @@ function runOnAllBrokers()
 {
   namespace=$(getNamespace)
 
-  pods=$(kubectl get pod -n "$namespace" $(getBrokerLabels) -o jsonpath="{.items[*].metadata.name}")
+  pods=$(kubectl get pod -n "$namespace" "$(getBrokerLabels)" -o jsonpath="{.items[*].metadata.name}")
```

Comment:
:x: this quoating doesn't work here. We want to have multiple labels here, splitted. Either we ignore this shellcheck error or we implement this differently.

Reply:
Oh, of course :facepalm: I'll add an ignore instead.